CMakeLists.txt

```diff
@@ -42,8 +42,15 @@ include( ProjectSettings.cmake )
 if( NOT DEFINED BOOST_SETUP_CMAKE_FILE )
   set( BOOST_SETUP_CMAKE_FILE FrameworkInternals/BoostSetup.cmake )
 endif()
-message( STATUS "Setting up boost location and libraries using [${BOOST_SETUP_CMAKE_FILE}]")
-include( ${BOOST_SETUP_CMAKE_FILE} )
+if( EXISTS ${CMAKE_CURRENT_LIST_DIR}/${BOOST_SETUP_CMAKE_FILE} )
```

Comment:
Is this really necessary? Looks like a redundant fallback. The first one should be sufficient, or?

Reply:
in the windows CI build config file I do this
```
set( BOOST_SETUP_CMAKE_FILE HANDLED_INLINE_IN_BUILD_CONFIG )
```
So, I wanted the option that i could also tell the main CMakeLists.txt that there is no boost setup file to collect (boost is just resolved inline in that build config file)

Reply:
Then it is probably easier to just check a variable if the boost setup should be ignored or not and if it is ignored hint that the user must be dealing with this in the build config.